### ISSUE TITLE
Metrics start but not work

### ISSUE THREAD

**Issue by Evgeniy-Bondarenko** [openshift/origin#15599]
##### Version
```
$ oc version
oc v1.5.1+7b451fc
kubernetes v1.5.2+43a9be4
features: Basic-Auth GSSAPI Kerberos SPNEGO

openshift v1.5.1+7b451fc
kubernetes v1.5.2+43a9be4
```

##### Steps To Reproduce
1. Deploy over ansible 

##### Current Result
```
...
ALTER TABLE data_compressed WITH compaction = {
  'class': 'TimeWindowCompactionStrategy',
  'compaction_window_unit': 'DAYS',
  'compaction_window_size': '1'
}
--
2017-08-01 19:16:51,302 INFO  [org.cassalog.core.CassalogImpl] (metricsservice-lifecycle-thread) Applying ChangeSet
-- version: 6.1
ALTER TABLE data WITH compaction = {'class': 'SizeTieredCompactionStrategy'}
--
2017-08-01 19:16:51,414 INFO  [org.cassalog.core.CassalogImpl] (metricsservice-lifecycle-thread) Applying ChangeSet
-- version: 7.0

  CREATE TABLE metrics_expiration_idx (
    tenant_id text,
    type tinyint,
    metric text,
    time timestamp,
    PRIMARY KEY ((tenant_id, type), metric)
)

2017-08-01 19:16:51,552 INFO  [org.hawkular.metrics.api.jaxrs.MetricsServiceLifecycle] (metricsservice-lifecycle-thread) The keyspace hawkular_metrics is using a replication_factor of 1
2017-08-01 19:16:51,764 INFO  [org.hawkular.metrics.core.service.MetricsServiceImpl] (metricsservice-lifecycle-thread) HAWKMETRICS100001: Using a key space of 'hawkular_metrics'
2017-08-01 19:16:51,825 INFO  [org.hawkular.metrics.core.service.MetricsServiceImpl] (metricsservice-lifecycle-thread) HAWKMETRICS100008: Using max size of 2048 for string metrics
2017-08-01 19:16:51,825 INFO  [org.hawkular.metrics.core.service.MetricsServiceImpl] (metricsservice-lifecycle-thread) HAWKMETRICS100009: Using max number of retries 5 and max retry delay of 30000 ms for inserting data points
2017-08-01 19:16:51,834 INFO  [org.hawkular.metrics.core.service.MetricsServiceImpl] (metricsservice-lifecycle-thread) HAWKMETRICS100007: Using default data retention of 604800 seconds
2017-08-01 19:16:51,907 WARN  [org.hawkular.metrics.core.dropwizard.HawkularMetricsRegistryListener] (metricsservice-lifecycle-thread) Did not find meta data for openshift-oauth-kubernetes-response-time in the metric registry
2017-08-01 19:16:51,921 WARN  [org.hawkular.metrics.core.dropwizard.HawkularMetricsRegistryListener] (metricsservice-lifecycle-thread) Did not find meta data for openshift-oauth-latency in the metric registry
2017-08-01 19:16:52,318 WARN  [com.datastax.driver.core.Cluster] (cluster2-worker-1) Re-preparing already prepared query is generally an anti-pattern and will likely affect performance. Consider preparing the statement only once. Query='SELECT DISTINCT tenant_id, tname FROM metrics_tags_idx'
2017-08-01 19:16:52,342 INFO  [org.hawkular.metrics.core.jobs.JobsServiceImpl] (metricsservice-lifecycle-thread) Preparing to create and schedule COMPRESS_DATA job
2017-08-01 19:16:52,502 INFO  [org.hawkular.metrics.core.jobs.JobsServiceImpl] (metricsservice-lifecycle-thread) Created and scheduled JobDetailsImpl{jobId=65568695-cc69-4c47-9690-e09e3db97a4b, jobType=COMPRESS_DATA, jobName=COMPRESS_DATA, parameters={}, trigger=RepeatingTrigger{triggerTime=1501621200000, interval=7200000, delay=60000}, status=NONE}
2017-08-01 19:16:52,512 INFO  [org.hawkular.metrics.core.jobs.JobsServiceImpl] (metricsservice-lifecycle-thread) Preparing to create and schedule DELETE_EXPIRED_METRICS job
2017-08-01 19:16:52,525 INFO  [org.hawkular.metrics.core.jobs.JobsServiceImpl] (metricsservice-lifecycle-thread) Created and scheduled JobDetailsImpl{jobId=a81f6f7e-055c-48cc-bac7-270624900ef8, jobType=DELETE_EXPIRED_METRICS, jobName=DELETE_EXPIRED_METRICS, parameters={}, trigger=RepeatingTrigger{triggerTime=1501632000000, interval=604800000, delay=60000}, status=NONE}
2017-08-01 19:16:52,554 INFO  [org.hawkular.metrics.core.util.GCGraceSecondsManager] (metricsservice-lifecycle-thread) Checking tables in hawkular_metrics to see if gc_grace_seconds needs to be updated
2017-08-01 19:16:52,580 INFO  [org.hawkular.metrics.api.jaxrs.MetricsServiceLifecycle] (metricsservice-lifecycle-thread) HAWKMETRICS200005: Metrics service started
2017-08-01 19:16:52,581 INFO  [org.hawkular.metrics.core.util.GCGraceSecondsManager] (RxComputationScheduler-1) gc_grace_seconds for active_time_slices is set to 864000. Resetting to 0
2017-08-01 19:16:52,583 INFO  [org.hawkular.metrics.core.util.GCGraceSecondsManager] (RxComputationScheduler-1) gc_grace_seconds for cassalog is set to 864000. Resetting to 0
2017-08-01 19:16:52,583 INFO  [org.hawkular.metrics.core.util.GCGraceSecondsManager] (RxComputationScheduler-1) gc_grace_seconds for data is set to 86400. Resetting to 0
2017-08-01 19:16:52,767 INFO  [org.hawkular.metrics.core.util.GCGraceSecondsManager] (RxComputationScheduler-2) gc_grace_seconds for data_compressed is set to 864000. Resetting back to zero.
2017-08-01 19:16:52,851 INFO  [org.hawkular.metrics.core.util.GCGraceSecondsManager] (RxComputationScheduler-1) gc_grace_seconds for finished_jobs_idx is set to 864000. Resetting to 0
2017-08-01 19:16:52,946 INFO  [org.hawkular.metrics.core.util.GCGraceSecondsManager] (RxComputationScheduler-2) gc_grace_seconds for jobs is set to 864000. Resetting to 0
2017-08-01 19:16:53,020 INFO  [org.hawkular.metrics.core.util.GCGraceSecondsManager] (RxComputationScheduler-2) gc_grace_seconds for leases is set to 864000. Resetting to 0
2017-08-01 19:16:53,109 INFO  [org.hawkular.metrics.core.util.GCGraceSecondsManager] (RxComputationScheduler-1) gc_grace_seconds for locks is set to 864000. Resetting to 0
2017-08-01 19:16:53,216 INFO  [org.hawkular.metrics.core.util.GCGraceSecondsManager] (RxComputationScheduler-2) gc_grace_seconds for metrics_expiration_idx is set to 864000. Resetting to 0
2017-08-01 19:16:53,293 INFO  [org.hawkular.metrics.core.util.GCGraceSecondsManager] (RxComputationScheduler-1) gc_grace_seconds for metrics_idx is set to 864000. Resetting to 0
2017-08-01 19:16:53,383 INFO  [org.hawkular.metrics.core.util.GCGraceSecondsManager] (RxComputationScheduler-2) gc_grace_seconds for metrics_tags_idx is set to 864000. Resetting to 0
2017-08-01 19:16:53,484 INFO  [org.hawkular.metrics.core.util.GCGraceSecondsManager] (RxComputationScheduler-1) gc_grace_seconds for retentions_idx is set to 864000. Resetting to 0
2017-08-01 19:16:53,592 INFO  [org.hawkular.metrics.core.util.GCGraceSecondsManager] (RxComputationScheduler-2) gc_grace_seconds for scheduled_jobs_idx is set to 864000. Resetting to 0
2017-08-01 19:16:53,710 INFO  [org.hawkular.metrics.core.util.GCGraceSecondsManager] (RxComputationScheduler-1) gc_grace_seconds for sys_config is set to 864000. Resetting to 0
2017-08-01 19:16:53,810 INFO  [org.hawkular.metrics.core.util.GCGraceSecondsManager] (RxComputationScheduler-2) gc_grace_seconds for tasks is set to 864000. Resetting to 0
2017-08-01 19:16:53,886 INFO  [org.hawkular.metrics.core.util.GCGraceSecondsManager] (RxComputationScheduler-1) gc_grace_seconds for tenants is set to 864000. Resetting to 0
2017-08-01 19:16:54,172 INFO [org.hawkular.metrics.core.util.GCGraceSecondsManager] (RxComputationScheduler-2) Finished gc_grace_seconds updates in 1614 ms
```

but http or https ports are not listen/ not work

##### Expected Result

I want have working metrics

**Comment by mfojtik** [comment]
@jcantrill set as p1 for triage

**Comment by jcantrill** [comment]
@mwringe Do you see anything here?  It should be using the 1.5 deployer to install metrics.

**Comment by mwringe** [comment]
How exactly is this being installed? What metric options are specified in the inventory file?

**Comment by jcantrill** [comment]
This is being installed using the deployer since it is v1.5.  I am unable to find any issue.  I performed 'oc cluster up --metrics' and ran into some issues that I was able to work past:

* Initially deployment failed because it timed out before an image was pulled - prepulled the issue
* Metrics were not appearing in the web console - Had to accept the metrics URL (Known limitiation)

I do not see any problems with v1.5.1 metrics and 'oc cluster up'.  I am in favor of closing this issue unless @Evgeniy-Bondarenko can provide additional information to reproduce his issue.
